src/ORM/Tree/Tree.php

```diff
@@ -50,19 +51,34 @@ public function getRootNodes($rootAlias = 't')
      *
      * @return NodeInterface a node
      */
-    public function getTree($path = '', $rootAlias = 't')
-    {
-        $results = $this->getFlatTree($path, $rootAlias);
+    public function getTree(
+        $path = '',
+        $rootAlias = 't',
+        $materializedPathSeparator = null
+    ) {
+        $materializedPathSeparator = $materializedPathSeparator
+            ?? self::getMaterializedPathSeparator()
+        ;
+
+        $results = $this->getFlatTree($path, $rootAlias, $materializedPathSeparator);
 
         return $this->buildTree($results);
     }
 
-    public function getTreeExceptNodeAndItsChildrenQB(NodeInterface $entity, $rootAlias = 't')
+    public function getTreeExceptNodeAndItsChildrenQB(NodeInterface $entity, $rootAlias = 't', $materializedPathSeparator = null)
     {
-        return $this->getFlatTreeQB('', $rootAlias)
-            ->andWhere($rootAlias.'.materializedPath NOT LIKE :except_path')
+        $materializedPathSeparator = $materializedPathSeparator
+            ?? self::getMaterializedPathSeparator()
+        ;
+
+        $qb = $this->getFlatTreeQB('', $rootAlias, $materializedPathSeparator);
+
+        return $qb
+            ->andWhere($rootAlias.'.materializedPath NOT LIKE :except_exact_path')
+            ->andWhere($rootAlias.'.materializedPath NOT LIKE :except_parent_path')
```

Comment:
You add two `WHERE NOT LIKE` clauses here but you use the QB coming from `getFlatTreeQB` which adds a `WHERE (LIKE ... OR LIKE ...)` clause. It's a bit strange imho.